x/programs/simulator/common/types.h

```diff
@@ -57,9 +57,13 @@ typedef char *(*InsertStateCallback)(void *data, Bytes key, Bytes value);
 typedef char *(*RemoveStateCallback)(void *data, Bytes key);
 
 typedef struct {
-    // this ptr gives context to the callbacks
+    // this is a pointer to the state passed in from rust
+	// it points to the state object on the rust side of the simulator
```

Comment:
this is properly formatted on my IDE and when I download the file as well

Reply:
It's a hard tab instead of spaces